astropy/coordinates/tests/test_formatting.py

```diff
@@ -55,8 +56,23 @@ def test_to_string_decimal():
     assert angle3.to_string(decimal=True, precision=1) == "4.0"
     assert angle3.to_string(decimal=True, precision=0) == "4"
 
-    with pytest.raises(ValueError, match="sexagesimal notation"):
-        angle3.to_string(decimal=True, sep="abc")
+
+@pytest.mark.parametrize("sep", [":", ":.", "dms", "hms"])
```

Comment:
After your changes, this seems a bit of an excessive number of parametrized tests, but given the history perhaps not bad...

Reply:
Better safe than sorry ? 😅